src/webui/www/private/scripts/prop-peers.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
```

Comment:
you probably should add a license header here... but it will be served out to client too, taking unnecessary bandwidth.

Reply:
Seems our other prop js files don't have a license either

Reply:
Well it is hard to track the history to find who contributed it... @sledgehammer999 any comments about the license header?

Reply:
It seems there are 2 sets of .js files in the same folder. Some (older?) that have a MIT license and some that have no license at all.
I assume that if you have no license specified in the file then the general license of the project applies? aka GPLv2
Since you create this file, you have the liberty to assign it a less restrictive license like MIT if you want. Probably the bandwidth concerns can be mitigated due to the compression used for the data transfer. Or as a last resort have a line something like "Copyright by X person, license MIT".

Reply:
I've added our standard GPLv2 license to the file